src/core/unit.c

```diff
@@ -2327,6 +2328,7 @@ static int unit_log_resources(Unit *u) {
         char* message_parts[1 + 2 + 2 + 1], *t;
         nsec_t nsec = NSEC_INFINITY;
         uint64_t memory_peak = UINT64_MAX;
+        uint64_t memory_swap_peak = UINT64_MAX;
```

Comment:
Please merge the two variable declarations into single line.

Reply:
Done.